plugin/src/Reconciler.lua

```diff
@@ -27,6 +28,29 @@ function Reconciler:applyUpdate(requestedIds, virtualInstancesById)
 	end
 end
 
+function Reconciler:mapProperties(virtualProperties, virtualInstancesById)
+	-- Maps properties from the Rojo format to one rbx-dom-lua uses
```

Comment:
Ideally these should be the same, since Rojo's format is just `rbx_dom_weak`'s format round-tripped through JSON.

Reply:
Usually it is, only the hack for BinaryString and the thing used to get the instance from the Ref ID are different.